packages/hoprd/crates/hoprd-misc/src/cli.rs

```diff
@@ -434,6 +434,30 @@ pub struct CliArgs {
         env = "HOPRD_CONFIGURATION_FILE_PATH"
     )]
     pub configuration_file_path: Option<String>,
+
+    #[arg(
+        long = "safeTransactionServiceProvider",
+        value_name = "SAFE_TX_SERVICE_PROVIDER",
```

Comment:
All env vars should be prefixed with `HOPRD_`

Reply:
updated